src/main/kotlin/no/nav/helse/sakskompleks/SakskompleksProbe.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.helse.sakskompleks
+
+import io.prometheus.client.Counter
+import no.nav.helse.sakskompleks.domain.Sakskompleks
+import org.slf4j.LoggerFactory
+
+class SakskompleksProbe {
+
+    companion object {
+        private val log = LoggerFactory.getLogger(SakskompleksProbe::class.java)
+
+        val sakskompleksTotalsCounterName = "sakskompleks_totals"
+
+        private val sakskompleksCounter = Counter.build(sakskompleksTotalsCounterName, "Antall sakskompleks opprettet")
+                .register()
+    }
+
+    fun opprettetNyttSakskompleks(sakskompleks: Sakskompleks) {
```

Comment:
denne kunne hatt en enkel test som sjekka om counteren økte med 1 feks

Reply:
Samme som for inntektsmeldingProbe